guides/common/modules/proc_configuring-external-dns.adoc

```diff
@@ -13,17 +13,30 @@ To make any changes persistent, you must enter the `{foreman-installer}` command
 .Procedure
 
 ifndef::foreman-deb[]
-. Install the `bind-utils` package:
+. Unlock packages to enable installation of new packages:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} packages unlock
+----
+
+. Install BIND and the utility packages:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {package-install} bind bind-utils
 ----
+
+. Lock the packages:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} packages lock
+----
 endif::[]
 
 ifdef::foreman-deb[]
-
-. Install the `bind-utils` package:
+. Install BIND:
```

Comment:
You're right this was inconsistent, but I was a bit conflicted about this. I mostly didn't comment on it since we don't publish the guide and in master it's been rewritten to not have this section at all. But this is a good change.